src/vmware/azext_vmware/_params.py

```diff
@@ -23,12 +23,14 @@ def load_arguments(self, _):
     with self.argument_context('vmware private-cloud') as c:
         c.argument('cluster_size', help='Number of hosts for the default management cluster. Minimum of 3 and maximum of 16.')
         c.argument('internet', help='Connectivity to internet. Specify "Enabled" or "Disabled".')
+        c.argument('yes', help='Deletes without confirmation.')

```

Comment:
Please use the first person voice
```suggestion
        c.argument('yes', help='Delete without confirmation.')

```